packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -146,50 +208,96 @@ const useRootStyles = makeStyles({
   },
 });
 
+const useInputClassName = makeResetStyles({
+  gridColumnStart: '1',
+  gridColumnEnd: '2',
+  gridRowStart: '1',
+  gridRowEnd: '3',
+  outlineStyle: 'none',
+  border: '0',
+  padding: '0',
+  color: tokens.colorNeutralForeground1,
+  backgroundColor: 'transparent',
+  fontFamily: 'inherit',
+  fontSize: 'inherit',
+  fontWeight: 'inherit',
+  lineHeight: 'inherit',
+
+  '::placeholder': {
+    color: tokens.colorNeutralForeground4,
+    opacity: 1, // browser style override
+  },
+});
+
 const useInputStyles = makeStyles({
-  base: {
-    gridColumnStart: '1',
-    gridColumnEnd: '2',
-    gridRowStart: '1',
-    gridRowEnd: '3',
-    outlineStyle: 'none',
-    ...shorthands.padding(0),
+  disabled: {
+    color: tokens.colorNeutralForegroundDisabled,
+    cursor: 'not-allowed',
+    backgroundColor: tokens.colorTransparentBackground,
+    '::placeholder': {
+      color: tokens.colorNeutralForegroundDisabled,
+    },
   },
 });
 
-const useButtonStyles = makeStyles({
-  base: {
-    display: 'inline-flex',
-    width: '24px',
-    alignItems: 'center',
-    justifyContent: 'center',
-    ...shorthands.border(0),
-    position: 'absolute',
+const useBaseButtonClassName = makeResetStyles({
+  display: 'inline-flex',
+  width: '24px',
+  alignItems: 'center',
+  justifyContent: 'center',
+  border: '0',
+  position: 'absolute',
+
+  outlineStyle: 'none',
+  height: '16px',
+
+  backgroundColor: 'transparent',
```

Comment:
```suggestion
  backgroundColor: tokens.colorTransparentBackground,
```

Reply:
Ditto for the root being the default background color for buttons.